app/controllers/adjustments_controller.rb

```diff
@@ -51,7 +51,7 @@ def create
       flash[:notice] = "Adjustment was successful."
       redirect_to adjustment_path(@adjustment)
     else
-      flash[:error] = @adjustment.errors.collect { |model, message| "#{model}: " + message }.join("<br />".html_safe)
+      flash[:error] = @adjustment.errors.collect { |error| "#{error.attribute}: " + error.message }.join("<br />".html_safe)
```

Comment:
@seanmarcia the coverage report shows this is exercised in the spec run, but I don't think I see any `flash[:error]` expectations that seem to check this generated list.
- How does it look to you?
- Did I miss something?
- If it needs/deserves any (more?) testing (manual and/or automated), how?

Reply:
I think this is good enough!